CredentialProvider/Presenter/CredentialProviderPresenter.swift

```diff
@@ -44,6 +44,8 @@ class CredentialProviderPresenter {
                 switch action {
                 case .extensionConfigured:
                     self.view?.extensionContext.completeExtensionConfigurationRequest()
+                case .cancel:
```

Comment:
Have you tested that you can cancel this request and it stops the user from enabling autofill? I could be mistaken but I thought that `cancelRequest` was only for credential requests.

Reply:
Yeah it doesn't seem to do what I wanted.